shared/js/textChatView.jsx

```diff
@@ -202,6 +202,12 @@ loop.shared.views.chat = (function(mozL10n) {
       return (
         <div className={entriesClasses}>
           <div className="text-chat-scroller">
+            {loop.shared.utils.isDesktop() ? null :
+              <p className="welcome-message">
+                {mozL10n.get("rooms_welcome_text_chat_label",
+                  { clientShortname: mozL10n.get("clientShortname2") })}
+              </p>
+            }
```

Comment:
I've tested the patch locally and I can see both welcome messages. I think this lines shouldn't be added again.